packages/venia-concept/src/actions/purchaseHistory/actions.js

```diff
@@ -1,6 +1,13 @@
 import { createActions } from 'redux-actions';
 
 const prefix = 'PURCHASE_HISTORY';
-const actionTypes = ['SET_ITEMS', 'RESET', 'FETCH_PURCHASE_HISTORY_REQUEST'];
+const actionTypes = ['RESET'];
```

Comment:
This change is fine if we do a major version bump, but it sounds like that's not guaranteed. We'll wait to merge anything until our meeting today, but are redux action names typically considered API?

Reply:
Architects confirmed Redux Action Names will not be considered API inside of Venia, especially since this component isn't exposes.